maltego_trx/entities.py

```diff
@@ -35,3 +35,30 @@
 URL = "maltego.URL"
 Website = "maltego.Website"
 WebTitle = "maltego.WebTitle"
+
+
+def translate_legacy_properties(entity_type, v2_property):
```

Comment:
@phdowling here is the property translation

Reply:
Looks good at first glance, but let's define the dict globally in this file, not inline inside this function (not 100% sure but I think the way it's currently written the python interpreter will probably destroy and create this dict on every transform run)

Reply:
ack, fixed

Reply:
ack, fixed